### ISSUE TITLE
02 2F meaning

### ISSUE THREAD

**Issue by Yui-Huang** [paulhamsh/Spark-Bluetooth-Message-Format#1]
Hi

I think the 02 2F from app is request the Amp fw version

APP send ->  01 FE 00 00 53 FE 17 00 00 00 00 00 00 00 00 00 F0 01 01 00 02 2F F7
AMP send -> 01 FE 00 00 41 FF 1D 00 00 00 00 00 00 00 00 00 F0 01 01 6A 03 2F 01 4E 01 02 03 25 F7

and my amp fw version is 1.2.3.37 in hex is 01 02 03 25 are equal the last 01 02 03 25 F7

hope this will help

**Comment by paulhamsh** [comment]
Yes thank you! On mine it just responded 00 01 02 03 or something like that so I thought it was the hardware presets!
I will update the docs - thanks!
